openstack/datadog_checks/openstack/openstack.py

```diff
@@ -689,19 +689,29 @@ def get_stats_for_single_project(self, project):
         def _is_valid_metric(label):
             return label in PROJECT_METRICS
 
+        project_name = None
+        if 'name' in project:
+            project_name = project['name']
```

Comment:
these 3 lines could be `project_name = project.get('name')`

Reply:
that makes sense, thanks!